<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGetCircleLenWidget</class>
 <widget class="QWidget" name="CGetCircleLenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="lay_widget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>连接片方向</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmb_direction">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>上下</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>左右</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>胶带区域数量</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="spin_tapeNum">
           <property name="maximum">
            <number>9</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>圆心检测</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>圆环半径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_RingRadius">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>210</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>膨胀量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_dilation">
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>4500</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_circleDetect">
           <property name="text">
            <string>圆心检测</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>边缘线检测</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmb_quadrantPos">
           <item>
            <property name="text">
             <string>第一象限</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>第二象限</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>第三象限</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>第四象限</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_lineDetect">
           <property name="text">
            <string>直线检测</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_test">
        <property name="text">
         <string>检测</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_savePam">
        <property name="text">
         <string>保存参数</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_RingRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_circleDetect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_quadrantPos</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>btn_lineDetect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_dilation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_circleDetect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
